go-controller/pkg/ovn/controller/admin_network_policy/status.go

```diff
@@ -0,0 +1,104 @@
+package adminnetworkpolicy
+
+import (
+	"context"
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/util/retry"
+	"k8s.io/klog/v2"
+	anpapi "sigs.k8s.io/network-policy-api/apis/v1alpha1"
+)
+
+// Defined status.type fields for Admin Network Policy - This is prefixed with the zone name thus
+// creating one row per zone in the metav1.Condition array
+// NOTE: On every update of ANP, related pods and namespaces - if anything goes wrong this
+// this status type flaps between true and false. Users can use this to narrow down the malfunctioning zone
+// (DANGER): If this feature is used at 500-1000 node scale, then that many status rows will be created
+// and a 1000 controllers will be updating the same policy CRD  ¯\_(ツ)_/¯ -TODO: Test at least at 120 nodes.
+// (tssurya): Perhaps check with trozet and maybe do status update only for fresh creates?
+/* Sample Output ~~~~~~~~~~
+Status:
+  Conditions:
+    Last Transition Time:  2023-06-11T12:07:51Z
+    Message:               Setting up OVN DB plumbing was successful
+    Reason:                SetupSucceeded
+    Status:                True
+    Type:                  AdminNetworkPolicyReadyInZone-ovn-worker
+    Last Transition Time:  2023-06-11T12:07:51Z
+    Message:               Setting up OVN DB plumbing was successful
+    Reason:                SetupSucceeded
+    Status:                True
+    Type:                  AdminNetworkPolicyReadyInZone-ovn-control-plane
+    Last Transition Time:  2023-06-11T12:07:51Z
+    Message:               Setting up OVN DB plumbing was successful
+    Reason:                SetupSucceeded
+    Status:                True
+    Type:                  AdminNetworkPolicyReadyInZone-ovn-worker2
+Events:                    <none>
+*/
+const (
+	AdminNetworkPolicyReadyStatusType = "AdminNetworkPolicyReadyInZone-"
```

Comment:
ack I am fine with this suggestion, it makes sense since the CRD's type is ANP or BANP I don't need to tell the user that info on the status..
I like `Ready-In-Zone-` let's go with this..

Reply:
How do we feel though about having status for every node. Should we test this at scale?

Reply:
yes i badly want to do scale testing for this feature, i hope this gets prioritized in the next release as part of GA goals. I have a TODO comment on the top of this file to make sure we do this. but as of now in the tech preview phase this is not a priority :(